internal/backends/qt/qt_widgets/tablecolumn.rs

```diff
@@ -118,6 +116,17 @@ impl Item for NativeTableColumn {
             ascending as "bool",
             descending as "bool"
         ] {
+            QPainter *painter_ = painter->get();
+
+            #if defined(Q_OS_MAC)
+                QImage header_image(size, QImage::Format_ARGB32_Premultiplied);
+                header_image.fill(Qt::transparent);
+                {QPainter p(&header_image); QPainter *painter_ = &p;
+            #else
+                painter_->save();
+                auto cleanup = qScopeGuard([&] { painter_->restore(); });
+                painter_->translate(QPoint()); // There is bugs in the styles if the scrollbar is not in (0,0)
```

Comment:
I don't think we need this hack on other platform than mac.

Reply:
the code is removed